docs/specifications/tests/Consistency-TP/consistency06.md

```diff
@@ -0,0 +1,88 @@
+# CONSISTENCY06: SOA MNAME consistency
+
+## Test case identifier
+
+**CONSISTENCY06**
+
+## Objective
+
+All authoritative name servers must serve the same SOA record (section
+4.2.1) of [RFC 1034] for the tested domain. As per section 3.3.13 of 
+[RFC 1035] the MNAME field in the SOA RDATA refers to the name of 
+"the name server that was the original or primary source of data 
+for this zone". Inconsistency in MNAME of the domain might result in 
+operational failures for applications that uses MNAME.
+
+## Inputs
+
+* "Child Zone" - The domain name to be tested.
+
+## Ordered description of steps to be taken to execute the test case
+
+ 1. Obtain the set of name server IPs for the *Child Zone* from [Method4] 
+    and [Method5] ("Name Server IP").
+
+ 2. Create an SOA query for *Child Zone* apex.
+
+ 3. For each name server in *Name Server IP* do:
+
+    1. Send the query to name server.
+    2. If the name server does not respond with a DNS response, 
+       emit *[NO_RESPONSE]* for that name server and go to next server.
+    3. Retrieve the SOA RR from the response from the name server.
+    4. If the response does not include a SOA record in the ansser section
+       then emit *[NO_RESPONSE_SOA_QUERY]* for that server and go to next
+       server.
```

Comment:
How could you retrieve the SOA RR if there was none? Maybe put this step before the SOA RR retrieval step?

You said you'd made an update to address this, but maybe you forgot?

Reply:
Please review again.